cfgov/cfgov/settings/base.py

```diff
@@ -94,6 +94,7 @@
     'treemodeladmin',
     'housing_counselor',
     'hmda',
+    'diversity_inclusion'
```

Comment:
Don't forget to add this to the list of "first party" apps in [.isort.cfg](https://github.com/cfpb/cfgov-refresh/blob/69fd37433c06a0809158d71650125d72180b7333/.isort.cfg#L12). It's unfortunate that we have to maintain that list manually.

Reply:
Done in https://github.com/cfpb/cfgov-refresh/pull/5265/commits/a82f1df9e65bb0be103c0aeecfe68b70229b1de1.

Reply:
Done in a82f1df.